programs/server/Server.cpp

```diff
@@ -1816,6 +1816,9 @@ try
         try
         {
             global_context->loadOrReloadDictionaries(config());
+
+            if (config().getBool("wait_dictionaries_load_at_startup", false))
```

Comment:
Why do we use new setting instead of  existing `dictionaries_lazy_load`?

```
If `false`, all dictionaries are created when the server starts, if the dictionary or dictionaries are created too long or are created with errors, then the server boots without of these dictionaries and continues to try to create these dictionaries.
```

According to documentation it's expected that it loads dictionaries on startup?

Reply:
`dictionaries_lazy_load=0` starts loading dictionaries at startup
`wait_dictionaries_load_at_startup` makes sure dictionaries finished their loading at startup

Reply:
Got it, thanks, let's update a doc and we are good to go